src/pages/clients.tsx

```diff
@@ -10,6 +10,7 @@ import { MarkdownAst } from '../components/markdown/markdown-ast';
 import { LocalesQuery } from './index';
 import { useTranslation } from 'gatsby-plugin-react-i18next';
 import { SpaceShuttle } from '../components/icons/illustrations/space-shuttle';
```

Comment:
Here are 2 unused imports:

<img width="906" alt="Screenshot 2021-12-06 at 08 01 52" src="https://user-images.githubusercontent.com/7814926/144801986-65540856-4a20-4a25-8c68-a67e695a19f0.png">

Reply:
I learned to ignore the github stuff from the times we had the errors in our repo. I know that you cleaned that up so I promise to check them better. Thanks for the catch 👍